Vue Component.sublime-syntax.yaml-macros

```diff
@@ -19,64 +20,77 @@ contexts: !merge
         - meta_scope: meta.template.vue
         - match: '{{'
           scope: punctuation.definition.template.begin.html
-          embed: scope:source.js
-          embed_scope: source.js.embedded.vue
+          embed: scope:source.jsx
+          embed_scope: source.jsx.embedded.vue
```

Comment:
Why `jsx`?

Reply:
The readme includes directions for getting JSX highlighting in script tags to work. I assumed that this was because Vue supported JSX in script tags. I may have been mistaken.

Reply:
Script tags yes but this part seems to be about the code in interpolation blocks. Stuff like `<span>{{ f.x }}</span>`.
I don't think JSX is supported here. It wouldn't really make sense here.

Reply:
Good point; I've returned to JS for mustache templates and attributes.

Reply:
You might want to re-built the actual syntax after the change

Reply:
Done.